README.md

```diff
@@ -1,2 +1,91 @@
-# bender
-A Slack concourse CI resource that can trigger jobs
+# bender Resource
+
+A  Concourse resource that can trigger any job (deployments, releases, ...) using slack.
+
+## Source Configuration
+
+* `slack_token`: *Required.* The slack token to be used. For more info check [slack](https://api.slack.com/docs/token-types#bot)
+
+* `channel`: *Required.* The channel name to be used by the bot.
+
+* `bot_name`: *Optional.*, *default `bender`* The bot name will be used to identify and filter messages. All messages must be addressed to the bot `@bot_name some message`.
+
+* `grammar`: *Optional.* if not defined bender will respond to all mentions `@bot_name`. If grammar is defined bender will respond to messages matching the regex expression. Use [python regular expression](https://docs.python.org/2/library/re.html) syntax. See [examples](#Examples) for inspiration.
+
+* `template`: *Optional.* A string that will be evaluated and written to `template_filename` can be used as an input file for further jobs in the pipeline.
+
+* `template_filename`: *Optional.* *default `template_file.txt`* the file name for a generated template.
+
+## Behavior
+
+### `check`: Check for new messages that match the rules
+
+Check will report a new version `timestamp` if messages
+
+1: Are addressed to `@bot_name`.
+
+2: If `grammar` is defined it matches the regex of the message.
```

Comment:
I don't get meaning in this sentence. Maybe redo it like this: Check will report new version if any message in bot channel matches regular expression defined in `grammar`.